web/src/components/bgImagewithCard.js

```diff
@@ -37,33 +41,45 @@ export default function BgImagewithCard({
                 isTransparent={isTransparent}
                 isRight={isRight}
                 data={data}
-                isPrimaryDark={isPrimaryDark}
-                isSecondary={isSecondary}
                 isPrimary={isPrimary}
+                isPrimaryDark={isPrimaryDark}
                 isSecondaryDark={isSecondaryDark}
                 hasButton={hasButton}
-                href={href}
-                to={to}
+                link={link}
+                externalLink={externalLink}
+                internalLink={internalLink}
             />
         </BackgroundImage>
     );
 }
 
 BgImagewithCard.defaultProps = {
-    to: '',
-    href: '',
+    isPrimary: false,
+    isPrimaryDark: false,
+    isSecondary: false,
+    isSecondaryDark: false,
+    isTransparent: false,
+    hasButton: false,
+    isDark: false,
+    isRight: false,
+    internalLink: false,
+    externalLink: false,
```

Comment:
Some of these should be true (I think `isPrimary` and `isRight`)

Reply:
I don't think so tbh. We have defaults (left, and no button at all) and we want to user to make them true to change the color of the button or align it to the right (left is default)

Reply:
So what happens if a user creates a card and doesn't select anything?

Reply:
it will be left, and there is no button color

Reply:
Where is that defined?